examples/sveltekit/src/lib/ThemeSwitch.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+	function switchTheme() {
+		const html = document.getElementsByTagName('html').item(0);
+		const isLightTheme = html.getAttribute('class') !== 'dark';
+		html.setAttribute('class', isLightTheme ? 'dark' : 'light');
+		html.setAttribute('style', `color-scheme: ${isLightTheme ? 'dark' : 'light'}`);
```

Comment:
Why not to use package? E.g. https://github.com/saadeghi/theme-change

Reply:
RIght now? I don't see it worth it, this solution seems to work just fine.

Reply:
It doesn't detect system theme

Reply:
Nice, right!